### PR TITLE
Add templates for S3 invoicing

### PR DESCRIPTION
**Pull request by davidgubler**
## Summary

* Add templates for S3 invoicing

### CODE REVIEW

**Comment by Kidswiss** [comment]
Could you maybe provide a PDF/Screenshot of a generated invoice? So that I can see what these templates actually do.

**Comment by davidgubler** [comment]
[account.report_invoice.pdf](https://github.com/vshn/appuio-odoo-adapter/files/9198597/account.report_invoice.pdf)

Here's a test invoice I've generated with these templates.

File: description_templates/s3-storage:c-appuio-cloudscale-lpg-2.gotmpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{{- template "_s3_storage" . -}}
+{{- define "_s3_storage" -}}
+S3 Storage Cloudscale
```
Comment (tobru) [inline description_templates/s3-storage:c-appuio-cloudscale-lpg-2.gotmpl:3]: Object Storage - Storage (cloudscale.ch)

File: description_templates/s3-traffic-out:c-appuio-cloudscale-lpg-2.gotmpl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{{- template "_s3_traffic_out" . -}}
+{{- define "_s3_traffic_out" -}}
+S3 Traffic Out Cloudscale
```
Comment (tobru) [inline description_templates/s3-traffic-out:c-appuio-cloudscale-lpg-2.gotmpl:3]: Object Storage - Traffic Out (cloudscale.ch)

**Review by tobru — CHANGES_REQUESTED** [review]
The description needs to be changed, please use "Object Storage" instead of "S3" as the product is called like that (S3 is the name of the AWS product for Object Storage).

**Comment by davidgubler** [comment]
Thanks @tobru , good point about "object storage". I'll have to change this everywhere.

File: description_templates/s3-requests:c-appuio-cloudscale-lpg-2.gotmpl [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{{- template "_s3_requests" . -}}
+{{- define "_s3_requests" -}}
+S3 Requests Cloudscale
```
[the remark this answers is not in the document]
Comment (davidgubler) [reply description_templates/s3-requests:c-appuio-cloudscale-lpg-2.gotmpl]: Fixed.

File: description_templates/s3-storage:c-appuio-cloudscale-lpg-2.gotmpl (hunk 1 above)
Comment (davidgubler) [reply description_templates/s3-storage:c-appuio-cloudscale-lpg-2.gotmpl]: Fixed

File: description_templates/s3-traffic-out:c-appuio-cloudscale-lpg-2.gotmpl (hunk 2 above)
Comment (davidgubler) [reply description_templates/s3-traffic-out:c-appuio-cloudscale-lpg-2.gotmpl]: Fixed

**Comment by davidgubler** [comment]
I've renamed everything from "S3" to "Object Storage".

New generated invoice PDF:

[account.report_invoice.pdf](https://github.com/vshn/appuio-odoo-adapter/files/9206679/account.report_invoice.pdf)

**Review by tobru — APPROVED** [review]
Example invoice LGTM

**Pull request merged**